src/pages/home/HomePage.js

```diff
@@ -209,6 +209,9 @@ class App extends React.Component {
             ? styles.hamburgerOpenAbsolute : styles.hamburgerOpen;
         const visibility = !this.state.isHamburgerEnabled || this.props.isSidebarShown ? styles.dFlex : styles.dNone;
         const appContentWrapperStyle = !this.state.isHamburgerEnabled ? styles.appContentWrapperLarge : null;
+        const flexDirection = this.props.isSidebarShown
```

Comment:
I'm curious why we need to check if the Sidebar is shown here?

Reply:
Basically the chat content header is always going to behave like a flexRow - it might have:
- Hamburger icon (optional)
- Chat participants
- Pin icon

![image](https://user-images.githubusercontent.com/2319350/103811550-ed401c80-5011-11eb-98c0-e914a727c78c.png)

Reply:
Hi @deetergp,

Thanks for your review. The conflict has been resolved. 

Hi @shawnborton 

Thanks for the mockup and feedback.

Although we define the text `numberOfLines={2}` This is what happened when we are not changing the `flex-direction` on the commit https://github.com/Expensify/Expensify.cash/pull/1156/commits/f5c36d9e0ab7d1adbf7df8ca1f4b6e2e1891e3c2

![image](https://user-images.githubusercontent.com/11847279/103862351-809c4f00-50f1-11eb-89c2-dd641405c006.png)
 

I've merged the upstream master and resolved some conflicts. Luckily we don't need to change the `flex-direction` anymore with this latest upstream master.

Reply:
Okay great, so just to confirm, all of the screenshots in the original comment are up to date and everything is working correctly without changing the flex direction?

Reply:
Hi @shawnborton 

The screenshot has been updated

Thanks for your feedback